spec/features/users_feature_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+# As a user
+# So that I can post pictures on Instagram
+# I would like to create my own account
+feature 'Users' do
+  scenario 'can sign up' do
+    visit '/'
+    click_button 'Sign up'
+    fill_in 'Name', with: 'Jessica Jones'
```

Comment:
In general it's important to refactor stuff like this to helper methods, otherwise your tests become extremely coupled with the UI. Have a look at this for some tips on how to do it: https://gist.github.com/ptolemybarnes/2dfda607b85d01e113b0